src/main/java/io/github/fusionflux/portalcubed/framework/gui/widget/TitleWidget.java

```diff
@@ -0,0 +1,21 @@
+package io.github.fusionflux.portalcubed.framework.gui.widget;
+
+import net.minecraft.client.gui.Font;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.client.gui.components.AbstractStringWidget;
+import net.minecraft.network.chat.Component;
+
+public class TitleWidget extends AbstractStringWidget {
```

Comment:
Why does this exist

Reply:
Because `StringWidget` has a text shadow, and titles in GUIs do not have them

Reply:
what does vanilla use for titles?